src/main/java/de/uni_tuebingen/ub/nppm/servlet/NewPasswordServlet.java

```diff
@@ -90,7 +90,14 @@ private void sendLink(HttpServletRequest request, HttpServletResponse response)
         //Check if E-Mail is reguistered in database
         boolean emailIsRegistered = BenutzerDB.hasEmail(email);
 
-        if (email != null && !email.equals("") && emailIsRegistered == true) {
+        if(email != null && emailIsRegistered == true)
+        {
+             if (email.equals(""))
```

Comment:
Das macht so ehrlich gesagt keinen Sinn:
- Wenn eine Mail in der Datenbank vorhanden ist sollte sie nicht leer sein.
- Umgekehrt: Wenn die Mailadresse null oder leer ist macht es keinen Sinn zu prüfen ob sie in der DB vorhanden ist